presto-native-execution/src/main/java/com/facebook/presto/Dummy.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto;
```

Comment:
@tdcmeehan : Just had some nits around naming : 

Does this need to be in com.facebook.presto namespace only ? Since all tests use namespace com.facebook.presto.nativeworker, it is okay to use the same here as well ?

Also, is it required to name this class Dummy, especially since its also in the com.facebook.presto namespace ? How about NexusDummy or NativeDummy ? If we change the namespace to com.facebook.presto.nativeworker, then the name Dummy suffices.

Reply:
It doesn't need to be in any particular package.  The location of the package or naming doesn't matter.  I actually would prefer for this to not be in the same package as our tests, as it might show up during autocomplete when writing test code.  It's following a convention set by Presto on Spark [here](https://github.com/prestodb/presto/blob/master/presto-spark-package/src/main/java/com/facebook/presto/Dummy.java).

Reply:
Thanks for the explanation Tim.